tools/saqc/saqc.xml

```diff
@@ -344,50 +319,73 @@
-              <option value="clearFlags">clearFlags: Assign UNFLAGGED value to all periods in field</option>
+              <option value="andGroup">andGroup: Logical AND operation for Flags</option>
+              <option value="clearFlags">clearFlags: Assign the flag UNFLAGGED to all timestamps</option>
               <option value="flagDummy">flagDummy: Function does nothing but returning data and flags</option>
               <option value="flagManual">flagManual: Include flags listed in external data</option>
-              <option value="flagUnflagged">flagUnflagged: Function sets a flag at all unflagged positions</option>
+              <option value="flagUnflagged">flagUnflagged: Assign a `flag` to all timestamps</option>
               <option value="forceFlags">forceFlags: Set whole column to a flag value</option>
-              <option value="orGroup">orGroup: Flag all values, if at least one of the given ``field`` values is already flagged</option>
-              <option value="propagateFlags">propagateFlags: Flag values before or after flags set by the last test</option>
-              <option value="transferFlags">transferFlags: Transfer Flags of one variable to another</option>
+              <option value="orGroup">orGroup: Logical OR operation for Flags</option>
+              <option value="propagateFlags">propagateFlags: Propagate already assigned flags along the date axis</option>
+              <option value="transferFlags">transferFlags: Transfer flags from one field to another</option>
             </param>
             <when value="andGroup">
-              <param argument="field" type="text" value="" label="Field" help="The name of the variable to process.">
-                <validator type="empty_field"/>
-              </param>
-              <param argument="group" type="text" optional="true" label="A collection of ``SaQC`` objects" help="Flag checks are performed on all ``SaQC`` objects based on the variables specified in ``field``. Whenever all monitored variables are flagged, the associated timestamps will receive a flag"/>
-              <param argument="target" type="text" optional="true" label="Target" help="The name of the variable to process."/>
-              <param argument="flag" type="float" value="255.0" optional="true" label="Any, optional The flag value the function uses to mark observations" help="Defaults to the ``BAD`` value of the translation scheme"/>
+              <param argument="field" type="data_column" label="field" help="Variable to process" data_ref="data" display="checkboxes" multiple="true"/>
+              <param name="group" type="data" optional="true" label="group" help="A collection of ``SaQC`` objects. Flag checks are performed on all ``SaQC`` objects based on the variables specified in ``field``. Whenever all monitored variables are flagged, the associated timestamps will receive a flag" format="csv" multiple="true"/>
```

Comment:
This has type `Sequence[SAQC]`. Not sure how SAQC objects are supposed to be given in SAQC config files? Are there examples?